### DISCUSSION TITLE
预告片无法播放

### DISCUSSION THREAD

**Discussion by xietege** [metatube-community/jellyfin-plugin-metatube#188] [Q&A]
已经是生成了预告片的文件了，但是播放还是显示没有兼容的流或者是直接闪退，请问如何解决这个问题

**Comment by kk77mgj** [comment]
可能是网络问题，没法访问预告片网站？试试手动访问网站可不可以打开，生成的实际是一条链接，没有下载到本地。

**Reply by xietege** [reply]
手动复制链接可以打开，在emby里面点播放就不行

**Reply by xjasonlyu** [reply]
emby播放预告，需要后端能够正确连接预告的地址，它是依赖后端播放的。
